### PR TITLE
Add a sender (using phone mic) and make the app more user-friendly.

### PR DESCRIPTION
**Pull request by Zopieux**
This embeds the 2-line PR #10.

Preview:

preview

### CODE REVIEW

**Comment by gavv** [comment]
Cool, thanks! @ortex could you please review it?

**Comment by gavv** [comment]
#5

**Comment by ortex** [comment]
> Cool, thanks! @ortex could you please review it?

yeah sure, I'll review it in upcoming days

File: app/src/main/java/org/rocstreaming/rocdroid/MainActivity.kt [hunk 1]
```diff
@@ -118,4 +253,34 @@ class MainActivity : AppCompatActivity() {
         }
         return ""
     }
+
+    private fun setButtonState(
+        thread: Thread?,
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/MainActivity.kt:258]: I guess it's better to change thread param to `alive: Boolean` and make this function return `void`

**Review by ortex — COMMENTED** [review]
Thanks for the PR! See a couple of comments below.

File: app/src/main/java/org/rocstreaming/rocdroid/MainActivity.kt [hunk 2]
```diff
@@ -1,47 +1,82 @@
...
-    private var thread: Thread? = null
+    private lateinit var requestPermissionLauncher: ActivityResultLauncher<String>
+    private lateinit var receiverIpEdit: EditText
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        val ipTextView: TextView = findViewById(R.id.ipTextView)
-        ipTextView.text = getIpAddresses()
+        volumeControlStream = AudioManager.STREAM_MUSIC
```
[the remark this answers is not in the document]
Comment (Zopieux) [reply app/src/main/java/org/rocstreaming/rocdroid/MainActivity.kt:36]: Ah, same here. I have no idea why. I tried a bunch of things. Happy to learn it's not my device (Huwawei) that's buggy or something. God I hate Android.

File: app/src/main/java/org/rocstreaming/rocdroid/MainActivity.kt (hunk 1 above)
Comment (ortex) [reply app/src/main/java/org/rocstreaming/rocdroid/MainActivity.kt:258]: I'll fix this later or feel free to send another PR

**Comment by gavv** [comment]
@Zopieux thanks!! If've added your screenshot to README, closed #5, and created #13.

**Pull request merged**